helm/gatling-benchmark/values.yaml

```diff
@@ -2,7 +2,7 @@
 # This is a YAML-formatted file.
 
 # Domain on which reports will be available
-domain: forgeops.com
+domain: .forgeops.com
```

Comment:
Hmm, back to the period before the domain?

Reply:
We discussed this with Lee and it would be good to align this to other products and use context /gatling or something similar as in other products.